docs/source/notes/autograd.rst

```diff
@@ -219,24 +219,44 @@ Autograd for Complex Numbers
 **What notion of complex derivative does PyTorch use?**
 *******************************************************
 
-PyTorch follows `JAX's <https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html#Complex-numbers-and-differentiation>`_
-convention for autograd for Complex Numbers.
+PyTorch uses the Wirtinger Calculus to compute gradients for functions with complex valued input and (or)
+output. PyTorch's autograd convention for complex numbers allows gradient values to be used in
+Gradient Descent with the same update rule as that in the case of real numbers. This convention works for
+optimization problems where the objective function is real valued.
 
-Suppose we have a function :math:`F: ℂ → ℂ` which we can decompose into functions u and v
-which compute the real and imaginary parts of the function:
+Suppose we have a function :math:`F: ℂ → ℂ`, which is a part of a :math:`G: ℂ → ℝ` function,
+and can be decomposed into functions :math:`u` and :math:`v` which compute the real and imaginary parts of the function:
 
     .. code::
 
         def F(z):
             x, y = real(z), imag(z)
-            return u(x, y) + v(x, y) * 1j
+            s = u(x, y) + v(x, y) * 1j
+            return s
```

Comment:
Just an idea. You start here directly with the assumption, that the chain rule is known.
Maybe it would be better for a beginner, when you start with the core assumption:
e.g.: Interpreted the complex number as two numbers and calculate the derivative:

 grad_x = dG / dx where G \in \mathcal{R}

 grad_y = dG / dy where G \in \mathcal{R}

Since complex numbers can contain 2 real numbers, we can define:

grad = grad_x + 1j * grad_y

or

grad = grad_x - 1j * grad_y

And this is known as Wirtinger Calculus.

(Note: Dropping the 1/2 in eq. (3.2) of https://arxiv.org/pdf/1701.00392.pdf is usually what NNs do. At the time of writing the paper we weren't careful, because this equation is never implemented)